TTS-Common-Control-Policy.md

```diff
@@ -16,20 +16,20 @@ All GSA employees and contracts are subject to these policies.
+The GSA Office of the Chief Information Security Officer, otherwise known as GSA Information Security, is responsible for ensuring that all employees and contracts are familiar and trained in these policies, and to ensure that appropriate procedures are established and tested. The TTS Technology Portfolio Director acts as the liaison between GSA Information Security and TTS products, including cloud.gov. The Cloud Operations team is responsible for all procedures and implementations within cloud.gov.
 
 ## Management commitment
 
-The GSA Chief Information Security Officer is ultimately responsible for these policies. 
+The GSA Chief Information Security Officer is ultimately responsible for these policies.
 
-## Coordination 
+## Coordination
 
-GSA Information Security works in close coordination and collaboration with all of the Services of the GSA, especially the [Technology Transformation Service (TTS)](https://www.gsa.gov/tts). 18F, an office of TTS, is home to multiple infrastructure and security experts. Implementations, and recommendations for policy additions and improvements, are constantly coordinated between the two teams. 
+GSA Information Security works in close coordination and collaboration with all of the Services of the GSA, including the [Technology Transformation Service (TTS)](https://www.gsa.gov/tts). TTS Technology Portfolio is home to multiple infrastructure and security experts, so implementations, and recommendations for policy additions and improvements, are constantly coordinated between GSA Information Security, TTS Technology Portfolio, and cloud.gov.
 
 ## Compliance
 
-The TTS Infrastructure Director is ultimately responsible for ensuring that the procedures of all TTS systems, including cloud.gov, adhere to these policies. The TTS Infrastructure Director accepts the risk of operating all Low and Moderate impact systems as the Authorizing Official, and reviews for compliance before granting an authority to operate (ATO).	
+The cloud.gov System Owner is ultimately responsible for ensuring that the procedures within cloud.gov adhere to these policies, and in ensuring cloud.gov maintains its authorization status with the FedRAMP JAB. The FedRAMP program management office (PMO) acts as the Authorizing Official for cloud.gov.
 
 ## Reviews
 
-GSA Information Security and TTS Infrastructure reviews all policies and procedures when new ATOs are required, or once every 3 years, whichever comes first. The teams also review modifications to policies and procedures when modifications become necessary due to changes in law, regulation, or operations.
\ No newline at end of file
+GSA Information Security and the TTS Technology Portfolio review all policies and procedures when new ATOs are required, or once every 3 years, whichever comes first. The teams also review modifications to policies and procedures when modifications become necessary due to changes in law, regulation, or operations.
```

Comment:
Again, this doesn't apply to cloud.gov in the same way, since I think GSA mostly defers to the JAB.

Reply:
Well, a lot of of overarching policy comes down through GSA and TTS, which then bubbles up through how cloud.gov operates, so it's close enough.